c7n/resources/rdscluster.py

```diff
@@ -110,6 +111,13 @@ def process(self, resources, event=None):
 RDSCluster.filter_registry.register('network-location', net_filters.NetworkLocation)
 
 
+@RDSCluster.filter_registry.register('kms-alias')
+class KmsAliasFilter(ResourceKmsKeyAlias):
```

Comment:
any reason why this subclass vs filters/kms.py ? i'm noticing this baseclass has some problems in that it fetches all keys instead of the ones of interest.

Reply:
Okie I'll see about swapping it out, I don't have a good reason beyond consistency with `ebs.filters.kms-alias`. If we tweak things here, is it worth revisiting the EBS filter as a follow-up?